### PR TITLE
Separate container responsibilty

### PR DESCRIPTION
**Pull request by teghnet**
[#160656547] A workable Docker Compose setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,24 @@
...
+```
+
+Update your /etc/hosts to include:
+
+```
+127.0.0.1 server.ads
+```
+
+Done. Now check:
+- http://server.ads/ for the the server
```
Comment (c3zi) [inline README.md]: type: `the the server`

File: bin/dev.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exec docker exec -it --user 1000 adshares_adserver_1 "$@"
```
Comment (c3zi) [inline bin/dev.sh:3]: Do we need first `exec`? :)

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  #      PHP__xdebug.dump.GET: '*'
+  #      PHP__xdebug.dump.POST: '*'
+  #      PHP__xdebug.dump.REQUEST: '*'
+  #      PHP__xdebug.dump.SERVER: '*'
+  #      PHP__xdebug.dump.SESSION: '*'
+  #      PHP__xdebug.dump_globals: '1'
+  #      PHP__xdebug.dump_once: '0'
+  #      PHP__xdebug.dump_undefined: '1'
+  #      PHP__xdebug.scream: '1'
+  #      PHP__xdebug.show_local_vars: '1'
```
Comment (c3zi) [inline docker-compose.yaml]: 1. Please remove all commented codes :)
2. There is a lot of options above. I'd suggest using `env_file` option. Then you can have all commented options locally (in the dedicated `.env` file) and global in `.env.dist`.

**Review by c3zi — CHANGES_REQUESTED** [review]

File: bin/dev.sh (hunk 2 above)
Comment (teghnet) [reply bin/dev.sh:3]: yes

File: docker-compose.yaml (hunk 3 above)
Comment (teghnet) [reply docker-compose.yaml]: :ok:

File: README.md (hunk 1 above)
Comment (teghnet) [reply README.md]: :ok:

**Review by c3zi — APPROVED** [review]

**Pull request merged**